pkg/registry/common/localbypass/find_server.go

```diff
@@ -26,16 +26,13 @@ type localBypassNSEFindServer struct {
 }
 
 func (s *localBypassNSEFindServer) Send(nse *registry.NetworkServiceEndpoint) error {
-	if nse.Url != s.nsmgrURL {
-		return s.NetworkServiceEndpointRegistry_FindServer.Send(nse)
-	}
 	if nse.ExpirationTime != nil && nse.ExpirationTime.Seconds == -1 {
 		return s.NetworkServiceEndpointRegistry_FindServer.Send(nse)
 	}
 
 	u, ok := s.nseURLs.Load(nse.Name)
 	if !ok {
-		return nil
+		return s.NetworkServiceEndpointRegistry_FindServer.Send(nse)
```

Comment:
Question... if we don't have the NSE in the cache... how exactly will we find its inode:// url ?  Looking a local NSE up in the registry doesn't help the NSMgr.

Reply:
This branch is needed for the remote usecase/interdomain.

Reply:
Could you explain why?  Are we not expiring nse.Name entries in localbypass?

Reply:
> Could you explain why? 

Yes, sure.

Actually, this server includes into nsmgr registry client :
https://github.com/networkservicemesh/sdk/blob/main/pkg/networkservice/chains/nsmgr/server.go#L214 (do not see that it is server, it is using as client if passed cc to registry 
 https://github.com/networkservicemesh/sdk/blob/main/pkg/networkservice/chains/nsmgr/server.go#L173)
 
So when nsmgr tries to resolve NSE that located on the second node or domain it is using this client  https://github.com/networkservicemesh/sdk/blob/main/pkg/networkservice/chains/nsmgr/server.go#L173.